docs/en_US/TrainingService/PaiMode.md

```diff
@@ -3,7 +3,33 @@
 NNI supports running an experiment on [OpenPAI](https://github.com/Microsoft/pai) (aka pai), called pai mode. Before starting to use NNI pai mode, you should have an account to access an [OpenPAI](https://github.com/Microsoft/pai) cluster. See [here](https://github.com/Microsoft/pai#how-to-deploy) if you don't have any OpenPAI account and want to deploy an OpenPAI cluster. In pai mode, your trial program will run in pai's container created by Docker.
 
 ## Setup environment
-Install NNI, follow the install guide [here](../Tutorial/QuickStart.md).
+Step 1. Install NNI, follow the install guide [here](../Tutorial/QuickStart.md).  
```

Comment:
do we need to point out that the setup steps is for paiyarn or paiK8s?

Reply:
No, this doc is only for paik8s, do not need to point out again. paiYarn has another doc `paiYarnMode.md`.